release/ray_release/test_automation/release_state_machine.py

```diff
@@ -74,6 +75,8 @@ def _create_github_issue(self) -> None:
         ]
         if not self.test.is_stable():
             labels.append("unstable-release-test")
```

Comment:
should we also give `"unstable-release-test"` a constant?

Reply:
let's do that!